include/nccl_ofi_param.h

```diff
@@ -239,18 +239,18 @@ OFI_NCCL_PARAM_INT(disable_dmabuf, "DISABLE_DMABUF", 0);
 OFI_NCCL_PARAM_UINT(round_robin_threshold, "ROUND_ROBIN_THRESHOLD", (256 * 1024));
 
 /*
- * Minimum bounce buffers posted per endpoint. The plugin will attempt to post
+ * Minimum bounce buffers posted per rail. The plugin will attempt to post
  * more bounce buffers if we dip below this threshold, allocating new bounce
  * buffers if needed.
  */
-OFI_NCCL_PARAM_INT(rdma_min_posted_bounce_buffers, "RDMA_MIN_POSTED_BOUNCE_BUFFERS", 64);
+OFI_NCCL_PARAM_INT(rdma_min_posted_bounce_buffers, "RDMA_MIN_POSTED_BOUNCE_BUFFERS", 16);
```

Comment:
with this split, there should be another set of parameters so that control messages (where 16 seems super small) and eager (where 16 is probably reasonable) are different.

Reply:
Done

Reply:
> I don't actually see that change?

See `rdma_min_posted_ctrl_recv_buffers` and `rdma_max_posted_ctrl_recv_buffers` in param.h.

Reply:
> ah, no, you need to reorder the commits, or merge them. But you can't have a commit where you radically shrink the bounce buffer size, only to change the meaning of the parameters in the next commit. Remember, there's no grouping of commits into PRs in the git history.

Sure, I'll merge them. But the first commit in this PR (as it is now) doesn't change the number of posted buffers (assuming `num_rails=4`). Before this PR, we divided the parameter by the number of rails, but the first commit just makes the parameter a per-rail value and divides the default value by 4.

Reply:
Merged